src/cart/src/include/cart/api.h

```diff
@@ -754,13 +754,14 @@ typedef int (*crt_rpc_task_t) (crt_context_t *ctx, void *rpc_hdlr_arg,
  *
  * \param[in] crt_ctx          The context to be registered.
  * \param[in] rpc_cb           The RPC process callback.
+ * \param[in] iv_reps_cb       The IV response callback.
  * \param[in] arg              The argument for RPC process callback.
```

Comment:
this should be then 'The argument for RPC and IV response callback' and structure internally renamed to some such like crt_crtx->cc_cb_arg

Reply:
yes, let's fix it in later cleanup patches.